versioning/scripts/dockerfiles/main.go

```diff
@@ -15,14 +15,55 @@ import (
 	"text/template"
 
 	"github.com/GoogleCloudPlatform/runtimes-common/versioning/versions"
+	"strconv"
 )
 
+type Format struct {
```

Comment:
Can we add some doc to this struct? It's not obvious what `Count` is for

Reply:
Removed count, merging it's function with 'indent_type' as 'indent'.